spec/unit/worker_spec.rb

```diff
@@ -68,7 +68,7 @@ class JobClass; end
         job.should respond_to(:fail).with(2).arguments
         job.should_receive(:fail) do |group, message|
           last_line = message.split("\n").last
-          expect(last_line).to match(/base\.rb:\d+:in `around_perform'/)
+          expect(last_line).to match(/base\.rb:\d+:in `perform'/)
```

Comment:
WDYT about extracting the regex into a variable named `first_line_in_perform_method_regex` or similar?  Then it's explicitly labeled.